.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,15 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
.idea should be in .gitignore

Reply:
this look quite "personal", there's e.g.a kotlin section which we shouldn't mandate or overwrite in any way :) also format-on-save is a personal choice. If we are to enforce scalafmt formatting, this should be done in CI. Isn't the whole `.idea` in `.gitignore`?

Reply:
Ok, I'll add whole `.idea` to `.gitinngore` and remove these files.